hooks/useProposals.ts

```diff
@@ -89,13 +91,20 @@ const createFetcher =
         const bridgedVotes = voteBridgedLogs?.find(
           (event) => event.args.proposalId?.toString() === proposalId
         )?.args;
-        const [againstVotes, forVotes, abstainVotes] = await getProposalVotes(BigInt(proposalId));
+        const proposalVotePromise = await getProposalVotes(BigInt(proposalId));
```

Comment:
if we want this to be a promise:

```suggestion
        const proposalVotePromise = getProposalVotes(BigInt(proposalId));
```

Reply:
https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/172/commits/60f4a367b6f16140a8e1f36423fbc8020a52dacd